src/eredis_client.erl

```diff
@@ -61,7 +61,11 @@
 -spec start_link(Options::options()) ->
           {ok, Pid::pid()} | {error, Reason::term()}.
 start_link(Options) ->
-    gen_server:start_link(?MODULE, Options, []).
+    case proplists:lookup(name, Options) of
+        {name, Name} ->
+            gen_server:start_link(Name, ?MODULE, Options, []);
+        none -> gen_server:start_link(?MODULE, Options, [])
```

Comment:
Either linebreak after both arrows or no linebreak after the arrows, but both same.

```suggestion
        {name, Name} ->
            gen_server:start_link(Name, ?MODULE, Options, []);
        none ->
            gen_server:start_link(?MODULE, Options, [])
```

Reply:
Done 👍